### PR TITLE
Update lang_zh_CN.json

### PR DESCRIPTION
**Pull request by Rain-Clear**
This update have some improvements regarding Chinese Simplified Chinese translation.

Thank you for your contribution to the Sandboxie repository.

In order to reduce the risk of accidental merges, it's highly recommended for beginners to open a new Pull Request in draft state by clicking the button "Create Draft Pull Request", instead of using the default "Create Pull Request" option. 

In addition, you can convert an existing pull request to a draft by clicking the "Convert to draft" link in the right sidebar under "Reviewers".

All new translators are encouraged to look at the "Localization notes and tips" before creating a pull request: https://git.io/J9G19

### CODE REVIEW

**Comment by love-code-yeyixiao** [comment]
If you finished your work,you could make the request "Ready for review".

File: SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
 "Select which browser you want to create a shortcut for and in what box": "选择要在哪个沙盒中为哪个浏览器创建快捷方式",
 "Select Browser Path": "选择浏览器路径",
 
-"Explorer Context Menu extension does not work": "Windows 资源管理器上/下界面菜单扩展选项（按钮/右键菜单）不工作",
+"Explorer Context Menu extension does not work": "Windows 资源管理器上/下界面菜单的扩展选项（例如按钮/右键菜单）不工作",
```
Comment (love-code-yeyixiao) [inline SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:26]: 将“上/下界面"改为"上下文"或许会更符合国内的流行翻译。

File: SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json [hunk 2]
```diff
@@ -53,20 +53,20 @@
 "Start/Run restrictions are in effect for the sandbox in which the program is running. The program is prohibited from starting or running.": "“启动/运行限制” 在运行该程序的沙盒被开启，该程序被禁止启动或运行。",
 
 "Failures to load SbieDll.dll when encountered by Chrome, or another software employing app containers, it is often caused by invalid DACL entries for the Sandboxie home directory. This mitigation measure will fix them, for this reason it will prompt for admin privileges which need to be granted for kmdutil.exe.": "Chrome 或其他使用应用容器的软件遇到无法加载 SbieDll.dll 时，通常是由 Sandboxie 主目录的无效 DACL 条目引起的。此解决措施将修复它们，这将提示需要为 kmdutil.exe 授予管理员权限。",
-"The DACLs have been adjusted, please try to run your application again and indicate if the issue has been resolved.": "DACL 条目 已调整，请尝试再次运行应用程序，并指示问题是否已解决。",
+"The DACLs have been adjusted, please try to run your application again and indicate if the issue has been resolved.": "“DACL 条目” 已调整，请尝试再次运行应用程序，并指示问题是否已解决。",
```
Comment (love-code-yeyixiao) [inline SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:56]: ”指示“改为”关注“会令人更容易理解。

File: SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json [hunk 3]
```diff
@@ -53,20 +53,20 @@
...
+"Sandboxie was not able to execute one of its own programs. Check access permissions to the Sandboxie installation folder and/or reinstall Sandboxie.\n\nPossible Causes:\n1. Sandboxie was configured to block access to the folder containing its program files.\n2. A third-party (HIPS) security software was configured to block the execution of the program mentioned in the message.\n\nYou can submit an issue report on the next page to help us to analyze the issue.": "Sandboxie 无法执行自己的程序之一。检查对 Sandboxie 的安装文件夹 是否有访问权限和/或重新安装 Sandboxie。\n\n可能的原因:\n1. Sandboxie 被配置为阻止对包含其程序文件的文件夹的访问。\n2. 配置了第三方(HIPS)安全软件来阻止消息中提到的程序的执行。\n\n您可以在下一页提交问题报告，以帮助我们分析问题。",
```
Comment (love-code-yeyixiao) [inline SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:61]: "Sandboxie 无法执行自己的程序之一。请检查当前账户对 Sandboxie 的安装文件夹 是否有访问权限，并且(或者)重新安装 Sandboxie。\n\n可能的原因:\n1. Sandboxie 被配置为阻止对包含其程序文件的文件夹的访问。\n2. 配置了第三方(HIPS)安全软件来阻止消息中提到的程序的执行。\n\n您可以在下一页提交问题报告，以帮助我们分析问题。"

File: SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json [hunk 4]
```diff
@@ -53,20 +53,20 @@
...
 
 "Do you want to open the box option dialog to change this preset?": "是否要打开沙盒选项对话框来更改此预设？",
 "No, it is fine as it is": "不，我觉得这样就好",
 "No, but I want to report a bug": "不，我要报告错误",
 
 "The message %1 is caused by the file migration limit being reached.": "消息 %1 是因为文件迁移请求达到限制引起的。",
 
-"It seems you are using a non-administrative user account on a system with enabled parental controls, this is known to be incompatible with Sandboxie.\nYou have the following options to resolve the issue.": "您似乎在启用了家长控制的系统上使用非管理员用户帐户，这对于 Sandboxie 显然不会兼容。\n您可以通过以下选项来解决此问题。",
+"It seems you are using a non-administrative user account on a system with enabled parental controls, this is known to be incompatible with Sandboxie.\nYou have the following options to resolve the issue.": "您似乎在启用了家长控制的系统上使用非管理员用户帐户，这对于 Sandboxie 显然不会被兼容。\n您可以通过以下选项来解决此问题。",
```
Comment (love-code-yeyixiao) [inline SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:69]: 这种情况下 Sandboxie 显然不会被兼容。

File: SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json (hunk 3 above)
Comment (RimacC2-EV) [reply SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:61]: 感谢您的批评指正！
Thanks!  :)

File: SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json (hunk 4 above)
Comment (RimacC2-EV) [reply SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:69]: 好的，我马上会更改它。
All right,I will change it as soon as possible.

File: SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json (hunk 3 above)
Comment (RimacC2-EV) [reply SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:61]: 感谢您的批评指正！
Thanks!  :)

File: SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json (hunk 2 above)
Comment (RimacC2-EV) [reply SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:56]: OK，我马上将修复它。感谢！：）
OK,I will fix it at once,Thanks! :)

Comment (RimacC2-EV) [reply SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:56]: 但我有一些个人对该翻译的理解，在我更改后您是否可以再次查看？
But I have some personal understanding of the translation, can you review it again after I change it?

Comment (love-code-yeyixiao) [reply SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:56]: 应该可以。
Maybe yes.

Comment (love-code-yeyixiao) [reply SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json:56]: > 但我有一些个人对该翻译的理解，在我更改后您是否可以再次查看？ But I have some personal understanding of the translation, can you review it again after I change it?

应该可以。
Maybe yes.

Comment (RimacC2-EV) [reply SandboxiePlus/SandMan/Troubleshooting/lang_zh_CN.json]: 非常感谢！！ :）
Thank you very much :)

**Review by love-code-yeyixiao — COMMENTED** [review]
I think that's OK.

**Comment by RimacC2-EV** [comment]
Thanks!!!

**Pull request merged**